### PR TITLE
Update windows docker to .net 4.8 and ltsc2022

### PR DESCRIPTION
**Pull request by larshg**

### CODE REVIEW

File: .dev/docker/windows/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # escape=`
 
-FROM mcr.microsoft.com/windows/servercore:ltsc2019
+FROM mcr.microsoft.com/dotnet/framework/runtime:4.8.1
```
Comment (larshg) [inline .dev/docker/windows/Dockerfile:3]:
```suggestion
FROM mcr.microsoft.com/windows/servercore:ltsc2022
```

File: .ci/azure-pipelines/env.yml [hunk 2]
```diff
@@ -107,14 +107,14 @@ jobs:
...
-    vmImage: 'windows-2019'
+    vmImage: 'windows-2022'
   strategy:
     matrix:
       Winx86:
         PLATFORM: x86
         TAG: winx86
         GENERATOR: "'Visual Studio 16 2019' -A Win32"
-        VCPKGCOMMIT: fd766eba2b4cf59c7123d46189be373e2cee959d
+        VCPKGCOMMIT: f6a5d4e8eb7476b8d7fc12a56dff300c1c986131
```
Comment (larshg) [inline .ci/azure-pipelines/env.yml:117]:
```suggestion
        VCPKGCOMMIT: 8eb57355a4ffb410a2e94c07b4dca2dffbee8e50
```

File: .ci/azure-pipelines/env.yml [hunk 3]
```diff
@@ -107,14 +107,14 @@ jobs:
   timeoutInMinutes: 360
   displayName: "Env"
   pool:
-    vmImage: 'windows-2019'
+    vmImage: 'windows-2022'
   strategy:
     matrix:
       Winx86:
         PLATFORM: x86
         TAG: winx86
```
Comment (larshg) [inline .ci/azure-pipelines/env.yml:115]:
```suggestion
        TAG: windows2022-winx86
```

File: .ci/azure-pipelines/env.yml [hunk 4]
```diff
@@ -107,14 +107,14 @@ jobs:
...
       Winx86:
         PLATFORM: x86
-        TAG: winx86
+        TAG: windows2022-x86
         GENERATOR: "'Visual Studio 16 2019' -A Win32"
-        VCPKGCOMMIT: fd766eba2b4cf59c7123d46189be373e2cee959d
+        VCPKGCOMMIT: 8eb57355a4ffb410a2e94c07b4dca2dffbee8e50
       Winx64:
         PLATFORM: x64
         TAG: winx64
```
Comment (larshg) [inline .ci/azure-pipelines/env.yml:120]:
```suggestion
        TAG: windows2022-x64
```

**Comment by larshg** [comment]
windows docker builds - I have added it as new tag under windows2022.

File: .dev/docker/windows/Dockerfile [hunk 5]
```diff
@@ -48,4 +48,6 @@ COPY $PLATFORM'-windows-rel.cmake' 'c:\vcpkg\triplets\'$PLATFORM'-windows-rel.cm
 
 RUN cd .\vcpkg;                                                `
     .\bootstrap-vcpkg.bat;                                            `
-    .\vcpkg install boost flann eigen3 qhull vtk[qt,opengl] gtest benchmark openni2 --triplet $Env:PLATFORM-windows-rel --host-triplet $Env:PLATFORM-windows-rel --clean-after-build;
+    .\vcpkg install boost flann eigen3 qhull vtk[qt,opengl] gtest benchmark openni2 --triplet $Env:PLATFORM-windows-rel --host-triplet $Env:PLATFORM-windows-rel --clean-after-build --x-buildtrees-root=C:\b; `
+    cat C:\b\qtdeclarative\install-$Env:PLATFORM-windows-rel-rel-out.log
```
Comment (mvieth) [inline .dev/docker/windows/Dockerfile:52]: This was probably for debugging, right? Let's remove it again?

**Review by mvieth — COMMENTED** [review]
I assume the MSVC CIs will pass as soon as the new images are pushed to the dockerhub? (after merging)

File: .dev/docker/windows/Dockerfile (hunk 5 above)
Comment (larshg) [inline .dev/docker/windows/Dockerfile:52]:
```suggestion
```

**Comment by larshg** [comment]
Yes, they should.

**Review by mvieth — APPROVED** [review]

**Pull request merged**